<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaPlayerWidget</class>
 <widget class="QWidget" name="MediaPlayerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(91, 91, 91, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1030</width>
     <height>555</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>5</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#ffffff;&quot;&gt;Media Player&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_date_time">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>5</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HomeButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../Images.qrc">
      <normaloff>:/Images/Home.png</normaloff>:/Images/Home.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Radio">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(81, 81, 81, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(255, 255, 255);
border-width: 1.2px;
border-style:inset;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../Images.qrc">
      <normaloff>:/Images/radio_icon_1.png</normaloff>:/Images/radio_icon_1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Menu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(81, 81, 81, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(255, 255, 255);
border-width: 1.2px;
border-style:inset;
color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>MENÃœ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Media">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(81, 81, 81, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(255, 255, 255);
border-width: 1.2px;
border-style:inset;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../Images.qrc">
      <normaloff>:/Images/music_icon_2.png</normaloff>:/Images/music_icon_2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Phone">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(81, 81, 81, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(255, 255, 255);
border-width: 1.2px;
border-style:inset;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../Images.qrc">
      <normaloff>:/Images/phone_icon_1.png</normaloff>:/Images/phone_icon_1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Navigation">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(81, 81, 81, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(255, 255, 255);
border-width: 1.2px;
border-style:inset;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../Images.qrc">
      <normaloff>:/Images/icon_navigation.png</normaloff>:/Images/icon_navigation.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="track_image_label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>231</width>
      <height>241</height>
     </rect>
    </property>
    <property name="pixmap">
     <pixmap resource="../Images.qrc">:/Images/Music_Lieder_Tawil.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_back">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>430</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 255, 255);
	border-width: 5px;
	border-radius: 30px;
}
QPushButton:pressed {
	border-color: rgb(0, 0, 0);
	border-width : 1.2px;
	border-style:inset;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../Images.qrc">
      <normaloff>:/Images/previous_icon.png</normaloff>:/Images/previous_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_next">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>430</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 255, 255);
	border-width: 5px;
	border-radius: 30px;
}
QPushButton:pressed {
	border-color: rgb(0, 0, 0);
	border-width : 1.2px;
	border-style:inset;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../Images.qrc">
      <normaloff>:/Images/next_icon.png</normaloff>:/Images/next_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_pause">
    <property name="geometry">
     <rect>
      <x>485</x>
      <y>430</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 255, 255);
	border-width: 5px;
	border-radius: 30px;
}
QPushButton:pressed {
	border-color: rgb(0, 0, 0);
	border-width : 1.2px;
	border-style:inset;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../Images.qrc">
      <normaloff>:/Images/play_icon.png</normaloff>:/Images/play_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="MuteButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>500</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 255, 255);
	border-width: 5px;
	border-radius: 10px;
}
QPushButton:pressed {
	border-color: rgb(0, 0, 0);
	border-width : 1.2px;
	border-style:inset;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../Images.qrc">
      <normaloff>:/Images/speaker_muted.png</normaloff>:/Images/speaker_muted.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>500</y>
      <width>381</width>
      <height>40</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
	border: 1px solid #bbb;
	background: white;
	height: 10px;
	border-radius: 5px;
}

QSlider::add-page:horizontal {
	background: #fff;
	border: 1px solid #777;
	height: 10px;
	border-radius: 4px;
}

QSlider::sub-page:horizontal {
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
   		stop: 0 rgba(255, 255, 255, 255), stop: 1 rgba(129, 0, 0, 255));
	background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
   		stop: 0 rgba(255, 255, 255, 255), stop: 1 rgba(129, 0, 0, 255));
	border: 1px solid #777;
	height: 10px;
	border-radius: 4px;
}

QSlider::handle:horizontal {
	background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    	stop:0 #eee, stop:1 #ccc);
	border: 2px solid #777;
	width: 16px;
	height: 20px;
	line-height: 20px;
	margin-top: -5px;
	margin-bottom: -5px;
	border-radius: 10px;
}

QSlider::handle:horizontal:hover {
	background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    	stop:0 rgba(72, 68, 71, 255), stop:1 rgba(255, 255, 255, 255));
	border: 2px solid #777;
	width: 16px;
	height: 20px;
	line-height: 20px;
	margin-top: -5px;
	margin-bottom: -5px;
	border-radius: 10px;
}

QSlider::sub-page:horizontal:disabled {
	background: #bbb;
	border-color: #999;
}

QSlider::add-page:horizontal:disabled {
	background: #eee;
	border-color: #999;
}

QSlider::handle:horizontal:disabled {
	background: #eee;
	border: 1px solid #aaa;
	border-radius: 4px;
}

QSlider{
	background-color: transparent;
}






</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="sliderPosition">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_volume">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>500</y>
      <width>71</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="progress_slider">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>340</y>
      <width>251</width>
      <height>40</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
	border: 1px solid #bbb;
	background: white;
	height: 10px;
	border-radius: 5px;
}

QSlider::add-page:horizontal {
	background: #fff;
	border: 1px solid #777;
	height: 10px;
	border-radius: 4px;
}

QSlider::sub-page:horizontal {
	background:  rgba(, 0, 0, 255);
	background-color: rgb(55, 166, 0);
	border: 1px solid #777;
	height: 10px;
	border-radius: 4px;
}

QSlider::handle:horizontal {
	background-color: rgb(231, 231, 231);
	border: 2px solid #777;
	width: 16px;
	height: 20px;
	line-height: 20px;
	margin-top: -5px;
	margin-bottom: -5px;
	border-radius: 10px;
}

QSlider::handle:horizontal:hover {
	background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    	stop:0 rgba(72, 68, 71, 255), stop:1 rgba(255, 255, 255, 255));
	border: 2px solid #777;
	width: 16px;
	height: 20px;
	line-height: 20px;
	margin-top: -5px;
	margin-bottom: -5px;
	border-radius: 10px;
}

QSlider::sub-page:horizontal:disabled {
	background: #bbb;
	border-color: #999;
}

QSlider::add-page:horizontal:disabled {
	background: #eee;
	border-color: #999;
}

QSlider::handle:horizontal:disabled {
	background: #eee;
	border: 1px solid #aaa;
	border-radius: 4px;
}

QSlider{
	background-color: transparent;
}






</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="sliderPosition">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_max_time">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>340</y>
      <width>71</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_current_time">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>340</y>
      <width>71</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_track_name">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>365</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: transparent;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Images.qrc"/>
 </resources>
 <connections/>
</ui>
